ui/proposals_page.go

```diff
@@ -72,27 +47,25 @@ type tabs struct {
 }
 
 type proposalsPage struct {
-	theme                      *decredmaterial.Theme
-	wallet                     *wallet.Wallet
-	proposalsList              *layout.List
-	tabs                       tabs
-	tabCard                    decredmaterial.Card
-	itemCard                   decredmaterial.Card
-	syncCard                   decredmaterial.Card
-	notify                     func(string, bool)
-	hasFetchedInitialProposals bool
-	isFetchingInitialProposals bool
-	legendIcon                 *widget.Icon
-	infoIcon                   *widget.Icon
-	selectedProposal           **dcrlibwallet.Proposal
-	state                      int
-	hasRegisteredListeners     bool
-	isSynced                   bool
-	updatedIcon                *widget.Icon
-	updatedLabel               decredmaterial.Label
-	syncButton                 *widget.Clickable
-	startSyncIcon              *widget.Image
-	timerIcon                  *widget.Image
+	theme            *decredmaterial.Theme
+	wallet           *wallet.Wallet
+	selectedProposal **dcrlibwallet.Proposal
+	proposals        **wallet.Proposals
+	proposalSync     *chan *wallet.NewProposal
```

Comment:
Is there any reason a pointer to the channel was used here?

Reply:
It is returning the proposal sync status and also the proposal from the proposal listener.